src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -399,7 +399,7 @@ void determineSpecificType(JSC::VM& vm, JSC::JSGlobalObject* globalObject, WTF::
         StringView view = str;
 
         const bool needsEllipsis = jsString->length() > 28;
-        const bool needsEscape = str->contains('"');
+        const bool needsEscape = str->contains('\'');
```

Comment:
why was this changed?

Reply:
node defaults to single quotes, and uses double quotes if a single quote is used in the string. I'll add a permalink

Reply:
we intentionally don't do it this way